www/README.md

```diff
@@ -0,0 +1,4 @@
+# Quickstart
```

Comment:
What does the www in the file path for these file stand for?

Reply:
www is just folder name to make it obvious that this is the frontend part of the project